frontend/packages/console-app/src/components/quick-starts/utils/quick-start-typings.ts

```diff
@@ -0,0 +1,23 @@
+import { QuickStartStatuses } from './quick-start-status';
+
+export type QuickStartItem = {
+  iconURL: string;
+  altIcon?: string;
+  id: string;
+  name: string;
+  duration: number;
+  description: string;
+  prerequisites?: string[];
+};
+
+export type QuickStartStatus = {
+  active?: boolean;
+  status: QuickStartStatuses;
+};
```

Comment:
Does this need to be a separate type. I cannot find this type being used anywhere else other than in the catalog item below. Merge it inline?

Reply:
updated